icons/paint-bucket.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m19 11-8-8-8.6 8.6a2 2 0 0 0 0 2.8l5.2 5.2c.8.8 2 .8 2.8 0L19 11Z" />
+  <path d="m19 11-8-8-8.6 8.6a2 2 0 0 0 0 2.8l5.2 5.2c.8.8 2 .8 2.8 0Z" />
```

Comment:
This path reaches outside our 1px padding box. My go-to method with 45deg corners is to just round them using a 2.41 radius which results in perfectly grid-aligned lines and corners:
```suggestion
  <path d="m19 11-8.3 8.3a2.4 2.4 0 0 1-3.4 0l-4.6-4.6a2.41 2.41 0 0 1 0-3.4L11 3Z" />
```